docs/operation/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,49 @@
+Bootstrap node backup restore
+=============================
+This section describes a reliable way for backup and restore a **MetalK8s**
+bootstrap node.
+
+Backup
+******
+
+A backup file is generated at the end of the bootstrap.
+
+To create a new backup file you just have to run the backup script
+
+.. code::
+
+    /srv/scality/metalk8s-X.X.X/backup.sh
+
+Backup archive are stored in `/var/lib/metalk8s/` directory
+
+Restore
+*******
+
+.. warning::
+
+    You can't use the restore script if you not have HA apiserver as SSH
+    informations to reconfigure all nodes are stored into apiserver.
+
+.. warning::
+
+    In case of 3 members etcd cluster (2 nodes + unreachable old bootstrap node)
+    you need to remove the old bootstrap node from the etcd cluster before
+    running the restore script.
+
+To restore a bootstrap node you need a backup file and **MetalK8s** ISOs.
+
+The ISOs need to be at the same place as the old bootstrap node, according to
+the `/etc/metalk8s/bootstrap.yaml` configuration file, so that all the versions
+will be automatically mounted at the right place.
+
+Like for a classic bootstrap you will need to mount an ISO and run the restore
```

Comment:
I think we can remove the reference to a classical bootstrap, just saying `First mount the ISO and then run the restore script`.